core/base-service/openapi.js

```diff
@@ -270,6 +272,27 @@ function category2openapi(category, services) {
           },
           example: 'violet',
         },
+        logoSize: {
+          name: 'logoSize',
+          in: 'query',
+          required: false,
+          description:
+            "Make icons adaptively resize by setting `auto`. It's useful for some wider logos like `amd` and `amg`.",
+          schema: {
+            type: 'string',
+          },
+          example: 'auto',
+        },
+        logoWidth: {
+          name: 'logoWidth',
+          in: 'query',
+          required: false,
+          description: 'The width of the logo, default to `14`',
+          schema: {
+            type: 'string',
+          },
+          example: '20',
+        },
```

Comment:
There has been some discussion in this PR about the `logoWidth` param, which already exists (as a hangover from the "popout" style) but is not documented. For this PR, lets maintain the current situation and not add it to the docs. Lets leave what we're doing about `logoWidth` for another day and remove this. Thanks.